### DISCUSSION TITLE
Overlap using GridLayout iOS issue

### DISCUSSION THREAD

**Discussion by sasabogi** [NativeScript/NativeScript#10508] [Q&A]
When two elements are overlapped using GridLayout, element below becomes unavailable because it is covered by another element. This is not the case with android. 
Code example:

 > <StackLayout>
    <GridLayout rows="*">
        <StackLayout row="0">
            <Button text="TEST"></Button>
        </StackLayout>
        <StackLayout row="0">
            <Button text="TEST1" style="margin-top: 50;"></Button>
        </StackLayout>
    </GridLayout>
  </StackLayout>

In this example the TEST button is not clickable on iOS, while on Android both are clickable.

**Comment by sasabogi** [comment]
Has anyone had this problem?  Please help

**Comment by rigor789** [comment]
The 2nd StackLayout fully covers the first one, and prevents interaction. 

You could move the margin to the StackLayout in this case, but the right approach depends on what you are trying to do.

**Reply by sasabogi** [reply]
I agree but on Android it also covers, but that doesn't prevent interaction. Whether that difference is natural behavior or not?

**Reply by rigor789** [reply]
I believe it's natural as the two platforms behave differently. I don't remember if it's for this purpose or not, but there is a `isPassThroughParentEnabled` property that might also work: https://docs.nativescript.org/api/class/LayoutBase#ispassthroughparentenabled
